spyder/utils/palette.py

```diff
@@ -78,6 +78,9 @@ class SpyderPaletteDark:
     SPYDER_LOGO_WEB = Logos.B40
     SPYDER_LOGO_SNAKE = Logos.B50
 
+    # Colors for disabled state toolbar buttons
+    COLOR_DISABLED = ICON_1, OPACITY_DISABLED
```

Comment:
You missed to prefix the `OPACITY_DISABLED` by the class that contains it. So please change this to

```python
COLOR_DISABLED = ICON_1, Opacity.OPACITY_DISABLED
```